extensions/PhabBugz/lib/Feed.pm

```diff
@@ -384,6 +391,41 @@ sub process_revision_change {
   );
   INFO($log_message);
 
+  # Change is a submission of the uplift request form.
+  if (_is_uplift_request_form_change($story_text)) {
+    my $revision_phid = $revision->phid;
+    INFO(
+      "Uplift request form submitted on $revision_phid, " .
+      "requesting `#release-managers` review."
+    );
+
+    # Get `#release-managers` review group.
+    my $release_managers_phid = Bugzilla::Extension::PhabBugz::Project
+      ->new_from_query({name => 'release-managers'})
+      ->phid;
+
```

Comment:
We may need some code here to make sure the new_from_query() was successful. In case the release-managers project does not yet exist or was removed. 

Should we skip the feed entry? Error out and keep trying until success? As it is written, if the new_from_query() does not find the project, and you try to access ->phid, it will crash.

Reply:
I changed this to just skip the message if `release-managers` doesn't exist.